client/styles/pages/view_proposal/index.scss

```diff
@@ -537,14 +537,16 @@ $discussion-title-font-size: 1.45rem;
             flex: 6;
             display: flex;
             flex-direction: row;
-            align-items: center;
+            align-items: left;
             position: relative;
         }
     
         .row-right {
```

Comment:
Little confused—there's no such corresponding class that exists in the codebase. 'row-right' only ever gets used in ListingRows, which are only ever owned by DiscussionRows. The Vote rows only have a 'row-left'